Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -13,4 +13,13 @@ extension OnlinePersistenceService: AnnotationsPersistence {
     func deleteAnnotation(annotation: Annotation) async -> Annotation? {
         await remotePersistence.annotations.deleteAnnotation(annotation: annotation)
     }
+
+    func createOrUpdateAnnotationsForLocal(annotations: [Annotation]) -> [Annotation]? {
+        guard let localAnnotationsPersistence = localPersistence.annotations as? LocalAnnotationsPersistence else {
```

Comment:
I think we can follow the naming in the other functions
```suggestion
        guard let localPersistence = localPersistence.annotations as? LocalAnnotationsPersistence else {
```